ark/analysis/visualize.py

```diff
@@ -19,44 +25,74 @@ def visualize_z_scores(z, pheno_titles):
+    )
+
+    # Sorts by Kagel Method :)
+    id_order = cell_data.groupby(patient_col_name).count().sort_values(
+        by=population_col_name,
+        ascending=False
+    ).index.values
+
+    pop_order = cell_data.groupby(population_col_name).count().sort_values(
+        by=patient_col_name,
+        ascending=False
+    ).index.values
+
     cell_data_stacked = cell_data_stacked.reindex(id_order, axis='index')
     cell_data_stacked = cell_data_stacked.reindex(pop_order, axis='columns')
 
     return cell_data_stacked
 
 
-def visualize_patient_population_distribution(cell_data, patient_col_name, population_col_name, color_map="jet",
-                                              show_total_count=True, show_distribution=True, show_proportion=True,
+def visualize_patient_population_distribution(cell_data, patient_col_name, population_col_name,
+                                              color_map="jet", show_total_count=True,
+                                              show_distribution=True, show_proportion=True,
                                               save_dir=None):
     """Plots the distribution of the population given by total count, direct count, and proportion
-              Args:
-                  cell_data (pd df): Dataframe containing columns with Patient ID and Cell Name
-                  patient_col_name (str): Name of column containing categorical Patient data
-                  population_col_name (str): Name of column in dataframe containing categorical Population data
-                  color_map (str): Name of MatPlotLib ColorMap used, default is jet
-                  show_total_count (bool): Boolean specifying whether to show graph of total population count, default is true
-                  show_distribution (bool): Boolean specifying whether to show graph of population distribution, default is true
-                  show_proportion (bool): Boolean specifying whether to show graph of total count, default is true
-                  save_dir (str): Directory to save plots, default is None"""
+
+    Args:
+        cell_data (pandas.DataFrame):
+            Dataframe containing columns with Patient ID and Cell Name
+        patient_col_name (str):
+            Name of column containing categorical Patient data
+        population_col_name (str):
+            Name of column in dataframe containing Population data
+        color_map (str):
+            Name of MatPlotLib ColorMap used. Default is jet
+        show_total_count (bool):
+            Boolean specifying whether to show graph of total population count, default is true
+        show_distribution (bool):
+            Boolean specifying whether to show graph of population distribution, default is true
+        show_proportion (bool):
+            Boolean specifying whether to show graph of total count, default is true
+        save_dir (str):
+            Directory to save plots, default is None
+
+    Returns:
+        None
```

Comment:
Do you prefer listing Returns None to just not having it here at all?

Reply:
I don't think I have a preference.  Both are fine.  Figure'd it'd be easier to clean up `Returns: None`, than to add them in later.

Reply:
it's more common to have nothing there, so I'll go with that